src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -159,6 +159,11 @@ public function firstOrFail($columns = array('*'))
 	 */
 	public function get($columns = array('*'))
 	{
+		if(is_string($columns))
```

Comment:
Close!  Space after `if` and before `(` :)  Be sure to squash to one commit once you get this fixed